recipe/meta.yaml

```diff
@@ -7,19 +7,10 @@ package:
 
 source:
   url: https://github.com/scipy/scipy/archive/v{{ version }}.tar.gz
-  sha256: 91269acb6d073f09743e8b6549fe5f8adb4329218a4e2c027074b54ba68c1e22
-  patches:
-    - gh10591.patch
-    - relax_gmres_error_check.patch  # [aarch64]
-    - skip_problematic_boost_test.patch  # [aarch64 or ppc64le]
-    - skip_problematic_root_finding.patch  # [aarch64 or ppc64le]
-    - skip_TestIDCTIVFloat_aarch64.patch  # [aarch64]
-    - skip_white_tophat03.patch  # [aarch64 or ppc64le]
-    # remove this patch when updating to 1.4
-{% if ver_major_minor == "1.3" %}
-    - gh11046.patch                # [ppc64le]
```

Comment:
Should(n't) the corresponding patch files be removed as well?

Reply:
Yup I forgot! Will do.